runtime/ms-rest-azure/package.json

```diff
@@ -26,7 +26,7 @@
   "license": "MIT",
   "dependencies": {
     "adal-node": "^0.2.2",
-    "async": "2.6.4",
+    "async": "^2.6.4",
```

Comment:
Can you also update the root package.json as well?

Reply:
updated. Thanks